examples/big_models_with_accelerate/README.md

```diff
@@ -0,0 +1,80 @@
+# Quantizing Big Models with HF Accelerate
+
+`llmcompressor` integrates with `accelerate` to support quantizing large models such as Llama 70B and 405B.
+
+## Overview
+
+[`accelerate`]((https://huggingface.co/docs/accelerate/en/index)) is a highly useful library in the Hugging Face ecosystem that supports for working with large models, including:
+- Offloading parameters to CPU
+- Sharding models across multiple GPUs with pipeline-parallelism
+
+
+### Using `device_map`
+
+To enable `accelerate` features with `llmcompressor`, simple insert `device_map` in `from_pretrained` during model load.
+
+```python
+from llmcompressor.transformers import SparseAutoModelForCausalLM
+MODEL_ID = "meta-llama/Meta-Llama-3-70B-Instruct"
+
+# device_map="auto" triggers usage of accelerate
+# if > 1 GPU, the model will be sharded across the GPUs
+# if not enough GPU memory to fit the model, parameters are offloaded to the CPU
+model = SparseAutoModelForCausalLM.from_pretrained(
+    MODEL_ID, device_map="auto", torch_dtype="auto")
+```
+
+`llmcompressor` is designed to respect the `device_map`, so calls to `oneshot` will work properly.
+
+### Practical Advice
+
+When working with `accelerate`, it is important to keep in mind that CPU offloading and naive pipeline-parallelism will slow down forward passes through the model. As a result, we need to take care to ensure that the quantization methods used fit well with the offloading scheme as methods that require many forward passes though the model will be slowed down.
+
+General rules of thumb:
+- CPU offloading should only be with data-free quantization methods (e.g. PTQ with `FP8_DYNAMIC`)
+- Multi-GPU is fast enough to be used with calibration data-based methods with `sequential_update=False`
+- It is possible to use Multi-GPU with `sequential_update=True`, but the runtime will be slower
```

Comment:
We should note that device_map="auto" does always work with GPTQ, because the automatic memory allocations won't take into account the hessian allocations for GPTQ. To take these into account we need to use the helper function:

```
device_map = calculate_offload_device_map(
    model_stub, reserve_for_hessians=True, num_gpus=2, torch_dtype=torch.float16
)
```

And this only works with sequential_update=True. Without this helper fn cpu offloading won't work correctly